javascript/README.md

```diff
@@ -93,3 +93,35 @@ const getResp = await momento.get(cacheName, cacheKey);
 // write file to disk
 fs.writeFileSync("./file-from-cache.txt", Buffer.from(getResp.bytes()));
 ```
+
+## Running the load generator example
+
+This repo includes a very basic load generator, to allow you to experiment with
+performance in your environment based on different configurations.  It's very
+simplistic, and only intended to give you a quick way to explore the performance
+of the Momento client running on a single nodejs process.
+
+Note that because nodejs javascript code runs on a single thread, the limiting
+factor in request throughput will often be CPU.  Keep an eye on your CPU
+consumption while running the load generator, and if you reach 100%
+of a CPU core then you most likely won't be able to improve performance further
```

Comment:
Maybe good to clarify `performance` vs `throughput`? They might actually start seeing poor response time performance when single core starts getting above 80% for example.

Reply:
sure can take a swing

Reply:
attempted to improve this in d423998132f04e68fb33060da3821826b95fc37c